crates/ide/src/hover/tests.rs

```diff
@@ -1310,6 +1310,42 @@ fn test_hover_function_show_qualifiers() {
     );
 }
 
+#[test]
+fn test_hover_function_show_types() {
```

Comment:
This function passed before my changes but I added it since there was no existing test which rendered a function with parameters.